pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/appengine/spark/SparkBootUtil.java

```diff
@@ -31,9 +30,9 @@ public class SparkBootUtil {
      *
      * @param stopRouteCallback an optional callback that is called
      *                          when the _ah/stop route is called by GAE
+     * @param cfg the config to use
      */
-    public static void startSparkServer(StopRouteCallback stopRouteCallback) {
-        Config cfg = ConfigManager.getInstance().getConfig();
+    public static void startSparkServer(StopRouteCallback stopRouteCallback, Config cfg) {
```

Comment:
Passing in config since DSM loads its config a bit differently from DSS and Housekeeping.